mgradm/cmd/install/shared/shared.go

```
@@ -32,6 +32,8 @@ func RunSetup(cnx *shared.Connection, flags *InstallFlags, fqdn string, env map[
 	}
 
 	err := adm_utils.ExecCommand(zerolog.InfoLevel, cnx, "/tmp/setup.sh")
+	_ = adm_utils.ExecCommand(zerolog.InfoLevel, cnx, "rm /tmp/setup.sh")
```

Why removing the error code here? Is this a dev leftover? and the error test after this would not behave as expected.

---

mm actually this was intentional:
- I removed `rm $0` from the setup script: in this way the return code of the setup script is the return code of `mgr-setup`
- I added this line to remove the script and we want to remove it because this action should be perform this action also if the setup fails, that's why it is performed before checking the error of the setup script
- I ignored the error in `rm /tmp/setup.sh` because I don't think the installation should fails if we have problems just on removing the script.

At least this was the intention :D do you have better idea to implement it?

---

why not storing the result of `mgr-setup` in a variable and return it at the end to keep the rm in the script?